src/ai/backend/manager/api/vfolder.py

```diff
@@ -2315,7 +2317,11 @@ async def delete_by_id(request: web.Request, params: DeleteRequestModel) -> web.
         folder_id,
     )
 
-    row = (await resolve_vfolder_rows(request, VFolderPermission.OWNER_PERM, folder_id))[0]
+    row = (
+        await resolve_vfolder_rows(
+            request, VFolderPermission.OWNER_PERM, folder_id, allow_privileged_access=True
+        )
+    )[0]
```

Comment:
If the number of vfolders must be exactly one, let's either raise an error or at least display a warning message when it isn't.

Reply:
Thats a good point, we should refactor all use case of this. But I am not going to change any other codes since this is a hotfix.
Please make a new issue for this

Reply:
Agreed, we only need to fix this in this PR.
I made a separate epic issue.
https://github.com/lablup/backend.ai/issues/3159